composer.json

```diff
@@ -52,6 +52,9 @@
     "wordpress-install-dir": "web/wp"
   },
   "scripts": {
+    "post-root-package-install": [
+            "php -r \"file_exists('.env') || copy('.env.example', '.env');\""
```

Comment:
Is the check for an existing file even necessary? Since this only happens on `create-project`, it's always a new project anyway.

This formatting is off as well, you need to match the indentation with the existing file.